README.md

```diff
@@ -30,14 +30,19 @@ To experience a Demo on your local machine run:
 npm start
 ```
 
-## Author
+## Author 1
 
 👤 **Tashobya Israel**
 
 - GitHub: [@githubhandle](https://github.com/tashisrael)
 - Twitter: [@twitterhandle](https://twitter.com/tashisrael)
 - Linkedin: [@linkedinhandle](https://www.linkedin.com/in/tashobya-israel-6a66b0181/l)
 
+👤 **Okonjo Gilbert**
+
+- GitHub: [@githubhandle](https://github.com/OpondoG)
+- Linkedin: [@linkedinhandle](https://www.linkedin.com/in/gilbert-okonjo-2081331b9/)
+
 ## 🤝 Contributing
 
```

Comment:
- [OPTIONAL] I noticed that the MIT/LICENSE Link is broken in your README, So I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯